mne/epochs.py

```diff
@@ -3682,16 +3729,16 @@ def _concatenate_epochs(epochs_list, with_data=True, add_offset=True, *,
                     (offsets[-1], len(out.ch_names), len(out.times)),
                     dtype=this_data.dtype)
             data[start:stop] = this_data
-    return (info, data, events, event_id, tmin, tmax, metadata, baseline,
-            selection, drop_log, verbose)
+    return (info, data, raw_sfreq, events, event_id, tmin, tmax, metadata,
+            baseline, selection, drop_log, verbose)
```

Comment:
when you concatenate epochs that have annotations, not only does the resulting epochs object lose those annotations, *it doesn't even have an `.annotations` property anymore*.  I think we should set it to `None` rather than having it missing entirely.  Not sure if that should happen here or in `_finish_concat`, do what you think makes sense.

Reply:
Okay, I've added a `BaseEpochs.set_annotations(None)` at the very end of BaseEpochs instantiation. That way, all `Epochs*` classes default to not having Annotations, unless they are instantiated outside of the `BaseEpochs.__init__()` call.